library/tinkerpop/pom.xml

```diff
@@ -115,4 +132,24 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>tinkerpop-standard-test</id>
+            <properties>
+                <!-- Gaffer doesn't pass all standard gremlin tests, this will hopefully improve with time -->
+                <maven.test.failure.ignore>true</maven.test.failure.ignore>
+                <test>GafferPopGraphStructureStandardTest,GafferPopFeatureTest</test>
```

Comment:
Has a ticket been created to address these later on?

Reply:
No yet, there's probably far too much for one ticket and I'm not sure we will ever be able to make them all pass due to how Gaffer fundamentally works.